pkgs/development/tools/mix2nix/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub, elixir, erlang }:
+
+stdenv.mkDerivation {
+  pname = "mix2nix";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "ydlr";
+    repo = "mix2nix";
+    rev = "b153a252f0689d3f6919e16c2df2fb3b6bcc9d2a";
+    sha256 = "15wyz7gdjywsqvsf1ad9gklvqsqi7lwgmyks3bjlc2sk6kpdxdmi";
+  };
+
+  buildInputs = [ elixir erlang ];
+
+  buildPhase = "mix escript.build";
```

elixir should just be needed for the build, but not runtime. Also will likely be more friendly to cross compilation situations.
```suggestion
  nativeBuildInputs = [ elixir ];
  buildInputs = [ erlang ];

  buildPhase = "mix escript.build";
```

---

Thanks for the review.

Adding to that, now that I think about it, erlang should not be needed at runtime as well